src/GameState.cpp

```diff
@@ -89,7 +90,13 @@ bool GameState::advanceFrame()
    m_time = SDL_GetTicks();
 
    long timePassed = std::min<long>(m_time - prevTime, GameState::MAX_FRAME_TIME);
-
+   
+   if(timePassed < GameState::MIN_FRAME_TIME)
+   {
+      long timeLeft = GameState::MIN_FRAME_TIME - timePassed;
+      SDL_Delay(timeLeft);
+   }
+   
    return step(timePassed);
```

Comment:
And now `timePassed` isn't actually how much time has passed. This needs fixing.